charts/monitoring-config/templates/prometheus-rules.yaml

```diff
@@ -10,6 +10,7 @@ spec:
 {{- range $path, $_ := .Files.Glob "rules/**.yaml" }}
 {{- if not (hasSuffix "_tests.yaml" $path) }}
   # {{ $path }}
-  {{- $.Files.Get $path | nindent 4 }}
+  # {{- $.Files.Get $path | replace "groups:" "" | nindent 2  }}
```

Comment:
Oh, just twigged these are YAML comments so we're still evaluating + outputting stuff from the Go/Helm template but just commenting out the first line of the template output 😅

Reply:
The syntax highlighting doesn't help here!